### PR TITLE
implemented simple database

### PR DESCRIPTION
**Pull request by unseeyou**
- Added a cog for user tag related commands
- Added a command to add a tag to a user
- Added example database

### CODE REVIEW

**Comment by unseeyou** [comment]
downside is that currently users have to add tags one at a time

File: .gitignore [hunk 1]
```diff
@@ -163,3 +163,7 @@ cython_debug/
 
 # VSCode
 .vscode/
+/bot/.env.old
+/tags.db
+/tags.db
+/tags.db
```
Comment (interrrp) [inline .gitignore:167]: Remove duplicates and `.env.old`

File: bot/exts/tag_commands.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "user-interfaces",
+]
+
+
+class Tags(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @slash_command(name="add_tag", description="Add a user tag to yourself")
+    async def ping(self, interaction: disnake.AppCommandInteraction, tag: TAGS) -> None:
```
Comment (interrrp) [inline bot/exts/tag_commands.py]: Why is the name `ping`? Just name it correctly and remove the decorator arg

**Review by interrrp — CHANGES_REQUESTED** [review]

File: bot/exts/tag_commands.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import typing
+
+import aiosqlite
+import disnake
+from disnake.ext.commands import Bot, Cog, slash_command
+
+TAGS = typing.Literal[
```
Comment (interrrp) [inline bot/exts/tag_commands.py]: Since this is a type, it should be named in PascalCase

File: bot/exts/tag_commands.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+class Tags(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @slash_command(name="add_tag", description="Add a user tag to yourself")
+    async def add_tag(self, interaction: disnake.AppCommandInteraction, tag: TAGS) -> None:
+        await interaction.response.send_message(f"Added tag `{tag}` to `{interaction.user.id}`")
+        tag = "!" + tag
+        conn = await aiosqlite.connect("tags.db")
```
Comment (interrrp) [inline bot/exts/tag_commands.py:40]: We should have a global connection to the database (see `bot.py`)

**Review by interrrp — CHANGES_REQUESTED** [review]

File: bot/exts/tag_commands.py (hunk 2 above)
Comment (unseeyou) [reply bot/exts/tag_commands.py]: fixed

File: bot/exts/tag_commands.py (hunk 4 above)
Comment (unseeyou) [reply bot/exts/tag_commands.py:40]: someone else do this bit I'm not sure how this will work

Comment (interrrp) [reply bot/exts/tag_commands.py:40]: assigned myself to the issue

**Review by interrrp — APPROVED** [review]

**Pull request merged**